menoh/mkldnn_with_generic_fallback/backend/generic/operator/mul.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef MENOH_IMPL_MKLDNN_WITH_GENERIC_FALLBACK_BACKEND_BACKEND_GENERIC_OPERATOR_MUL_HPP
+#define MENOH_IMPL_MKLDNN_WITH_GENERIC_FALLBACK_BACKEND_BACKEND_GENERIC_OPERATOR_MUL_HPP
+
+#include <menoh/array.hpp>
+#include <menoh/mkldnn_with_generic_fallback/procedure.hpp>
+
+namespace menoh_impl {
+    namespace mkldnn_with_generic_fallback_backend {
+        namespace generic_backend {
+            inline procedure make_mul(node const&,
+                                      std::vector<array> const& input_list,
+                                      std::vector<array> const& output_list) {
+                assert(input_list.size() == 2);
+                assert(output_list.size() == 1);
+
+                for(auto const& input : input_list) {
+                    if(input.dtype() != dtype_t::float_) {
+                        throw std::runtime_error("invalid dtype");
```

Comment:
Isn't it better to throw `menoh_impl::exception` with error code `menoh_error_code_invalid_dtype`?

Reply:
I'll fix it.